source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,94 @@
+ION Discover DJ Pro
+===================
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro
+
+Ion Discover DJ Pro is a 2-deck controller with integrated audio interface and iPhone dock.
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (schematic view)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (schematic view)
+
+-  `Mixxx User Forum <https://mixxx.discourse.group/t/ion-discover-dj-pro-mapping-script/12120>`__
+
+.. versionadded:: 2.2.5
+
+Compatibility
+-------------
+
+This controller is USB MIDI class compliant, so it is plug-and-play on all platforms without having to install
+any special drivers. The manufacturer’s page only mentions Windows and Mac but it’s confirmed working on Linux
+as well.
+
+Controller Mapping
+------------------
+
+.. note:: Only controls are documented that are known to be working.
+
+Deck section
+~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro_deck.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (deck section)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (deck section)
+
+===  ======================================================   ============================================================================================
+No.  Control                                                  Function
+===  ======================================================   ============================================================================================
+1    :hwlabel:`JOG WHEEL`                                     Scratches the track
+2    :hwlabel:`PLAY/PAUSE` button                             Plays/pauses a track in the respective Deck
+3    :hwlabel:`SYNC` button                                   Sync the tempo to other playing track
+4    :hwlabel:`SCRATCH` button                                Toggle the scratch mode
+5    :hwlabel:`TEMPO` slider                                  Adjust the track playing speed
+6    :hwlabel:`PITCH BEND` buttons -/+                        Lower (-) or Raise (+) the pitch temporarily
+7    :hwlabel:`REVERSE` button                                Hold for reverse play
```

Comment:
The button is labeled "Reverse Play" I think.

Reply:
Yes, you are right, I'll update this text.

Reply:
Done